HealthcareSystem/PatientWebAppTests/CreateObjectsForTests/StubRepository.cs

```diff
@@ -68,10 +68,14 @@ public IExaminationRepository CreateExaminationStubRepository()
         {
             var examinationStubRepository = new Mock<IExaminationRepository>();
 
+            Examination examinationValid = _objectFactory.GetExamination().CreateValidTestObject();
+            Examination examinationInvalid = _objectFactory.GetExamination().CreateInvalidTestObject();
```

Comment:
So, here's a thing that bothers me - these objects are actually both valid, it's just that one is cancellable and one is not. I would rename these methods to GetCancellableTestObject and GetUncancellableTestObject, or something like GetCreatedTestObject or GetCancelledTestObject, just so you know what they do later.

Reply:
Fixed in 8d297a7a201cb557715f22f218845f03c8793464.